packages/core/deploy/058_deploy_optimistic_governor.js

```diff
@@ -0,0 +1,33 @@
+require("dotenv").config();
+
+// Deploys the mastercopy of the OptimisticGovernor contract. It is not intended to be used directly, but rather to be
+// used when deploying minimal proxy contracts that delegate calls this mastercopy. Constructor arguments are arbitrary
+// here, just to satisfy the requirements of OptimisticGovernor constructor and make sure that the mastercopy is not
+// usable directly. Proxy contracts can be deployed using the ModuleProxyFactory from Gnosis Zodiac at
+// https://github.com/gnosis/zodiac/blob/master/contracts/factory/ModuleProxyFactory.sol and passing encoded bytes data
+// in its deployModule method's initializer parameter to call setUp on the deployed proxy contract.
+
+const func = async function (hre) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+
+  const Finder = await deployments.get("Finder");
+  const owner = "0x000000000000000000000000000000000000dEaD"; // Mastercopy contract should not be usable directly.
```

Comment:
is there a reason it is this address and not the `0x0` address?

Reply:
`transferOwnership` that is being called as part of constructor requires this to be non-zero address